tdesign/src/themes/muiTheme.ts

```diff
@@ -68,7 +68,12 @@ export const muiTheme = ({
      * Note: There is no constant including the nav links because their width is unknown
      */
     rightMarginInsideNavProfileMenu: `max(2rem, calc((100vw - ${breakpointValues.desktop}px) / 2) + ${magicNumbers.nav.profile.fromRightOfPageToLeftOfMenu})`,
+
     brandGradient: `linear-gradient(90deg, rgb(249 69 105 / 100%) 0%, rgb(255 128 153 / 50%) 100%)`,
+    bannerGradient: `linear-gradient(185.2deg, #FDF2F4 24.37%, rgba(253, 242, 244, 0) 103.81%)`,
+
+    paddedColumnLeftWidth: "max(25px, calc((100vw - 784px) / 2))",
+    paddedColumnRightWidth: "max(25px, calc((100vw - 784px) / 2))",
```

Comment:
I see these 2 constants are only used in a single component. I don't see why they should be defined as constants in the theme :smile: In my view, they should be defined as regular variables in the `PaddedSingleCol` component :slightly_smiling_face:

Reply:
Oh good point. I had a feeling that as more pages got added it on, it could be used more... but in the present, only used in one component!